Copy test output to stdout in addition to $log_file

--- a/test-driver	2021-10-04 00:20:29.356330263 -0300
+++ b/test-driver	2021-10-04 00:19:56.144741073 -0300
@@ -92,7 +92,7 @@
 trap "st=143; $do_exit" 15
 
 # Test script is run here.
-"$@" >$log_file 2>&1
+"$@" 2>&1 | tee $log_file
 estatus=$?
 if test $enable_hard_errors = no && test $estatus -eq 99; then
   estatus=1
